prisma/seed.ts

```diff
@@ -0,0 +1,101 @@
+import { faker } from "@faker-js/faker";
+import { PrismaClient } from "@prisma/client";
+import {
+  generateFakeDonation,
+  generateFakeDistributionsArray,
+  generateFakeDonor,
+  generateFakeTaxUnit,
+} from "./fakedata/generateFakedata";
+import { organizationsJSON } from "./fakedata/organizations";
+import { paymentsJSON } from "./fakedata/payments";
+import { distributionsJSON } from "./fakedata/distributions";
+
+const prisma = new PrismaClient();
+
+async function createFakeDonorWithFakeDonations() {
+  const fakeDonor = generateFakeDonor();
+
+  const fakeTaxUnit = generateFakeTaxUnit(fakeDonor.full_name);
+
+  const numberOfFakeDonations = faker.number.int({ min: 0, max: 10 });
+  const arrayOfFakeDonations = Array.from({ length: numberOfFakeDonations }).map(() =>
+    generateFakeDonation(),
+  );
+
+  const fakeDonorDB = await prisma.donors.create({
+    data: {
+      ...fakeDonor,
+      Donations: {
+        create: arrayOfFakeDonations,
+      },
+      Tax_unit: {
+        create: fakeTaxUnit,
+      },
+
+      //TODO: Virker som FundsDonations tabellen ikke brukes lengre
```

Comment:
Wondering if the FundsDonation Table is still in use? Or if it is any purpose to seed the table with testdata